BUILDING.md

```diff
@@ -46,6 +46,30 @@ make setup -j$(nproc) OPTFLAGS=-O2 DEBUG=0
 make -j $(nproc) OPTFLAGS=-O2 DEBUG=0
 ```
 
+## macOS
+
+1. Requires `gcc@12, sdl2, libpng, glew, spdlog, pulseaudio, x11` (can be installed via brew, etc)
```

Comment:
I could not find `x11` via brew so I install `libx11`

Reply:
That's correct but it's still x11, will leave this one as is. Not sure if it's called libx11 on other package managers.